File: crossingSum.py
def crossingSum(matrix, a, b):
    res=0
    max=0    
    for i in range(len(matrix)):   
        res+=matrix[i][b]    
        i+=1
    for j in range(len(matrix[0])):       
        if j==b:
            continue
        res += matrix[a][j] 
        j+=1      
    return res

File: java/crossingSum.java
int crossingSum(int[][] matrix, int a, int b) {

    int res=0, max=Integer.MIN_VALUE;    
   for (int i=0; i<matrix.length; i++)        
   {
       res+=matrix[i][b];
   }    
   
       for (int j=0; j<matrix[0].length; j++)        
       {    
           if(j==b) continue;
       res+=matrix[a][j];
       }    
       
   return res;
   

}